### DISCUSSION TITLE
Некоторым студентам не нужен ментор

### DISCUSSION THREAD

**Discussion by k1rsanov** [rolling-scopes-school/tasks#507] [Ideas]
**Проблема**
Некоторым студентам не нужен ментор. Они сами справляются с заданиями без дополнительной помощи и не общаются с менторами в процессе работы. Мне как ментору не хотелось бы тратить время на проверку заданий у таких людей.
Обычно у меня примерно один из пяти студентов заинтересован в получении опыта от ментора. Остальные формируют у меня ощущение траты моего времени

**Решение**
Было бы хорошо для решения этой проблемы давать студентам выбор, хотят ли они быть с ментором или нет. Давать поощрение в виде балов тем студентам, которые выбрали путь без помощи менторов.
Это сформирует у менторов пулл студентов, которые хотят получить в первую очередь знания от курса. Менторам с такими студентами будет более приятно работать. Ментор с большей вероятностью получит удовольствие от процесса, потому что он будет осознавать то, что он своими знаниями помогает студентам, и студентам эти знания нужны.

**Comment by CodeWorld-rss** [comment]
Отличная мысль, мне как студенту из категории пришедшего к Вам за знаниями, будет больше шансов заслужить-получить возможность менторства.

**Comment by natein** [comment]
Выскажусь в защиту студентов, которые выполняют задания самостоятельно. Это не значит, что им не нужен ментор, такие студенты  ничуть не меньше нуждаются в замечаниях по их коду, а также в замечаниях по реализации приложения. Например я ожидаю от ментора рекомендаций как приложение можно сделать лучше, указать на места, которые могут вызвать проблемы в дальнейшем.

**Reply by KirillDemyanenko** [reply]
Мне кажется и 1 из 5 менторов реально может что-то дать, в большинстве случаев это просто выставить дополнительную оценку. Если для того чтобы стать ментором нужно бы было выполнить какое-то даже минимальное требование, то менторов было бы раза в 2 меньше. Если отвязать оценки от наличия менторов, то это может работать, если наличие ментора будет напрямую означать более высокие оценки, то это работать не будет)
